drizzlepac/tweakreg.py

```diff
@@ -17,14 +17,13 @@
 
 from . import util
 
-# __version__ and __version_date__ are defined here, prior to the importing
-# of the modules below, so that those modules can use the values
-# from these variable definitions, allowing the values to be designated
+# __version__ is defined here, prior to the importing
+# of the modules below, so that those modules can use the value
+# from this variable definition, allowing the value to be designated
 # in one location only.
 #
 # This is specifically NOT intended to match the package-wide version information.
 __version__ = '1.4.7'
```

Comment:
I think we need to get rid of all `__version__`s defined per module. They are never updated when files change.

Reply:
While I agree, it's not the goal of this PR

Reply:
In fact, I have created a new issue, #1249?, for doing this based on a new Jira ticket [HLA-706](https://jira.stsci.edu/browse/HLA-706).